depends/README.md

```diff
@@ -118,7 +118,7 @@ The following can be set when running make: `make FOO=bar`
 - `NO_WALLET`: Don't download/build/cache libs needed to enable the wallet (SQLite)
 - `NO_BDB`: Don't download/build/cache BerkeleyDB
 - `NO_USDT`: Don't download/build/cache packages needed for enabling USDT tracepoints
-- `MULTIPROCESS`: Build libmultiprocess (experimental)
+- `NO_IPC`: Don't build experimental libmultiprocess (default on Windows and OpenBSD)
```

Comment:
In commit "build: depends makes libmultiprocess by default" (9b46b7d582edcf61bbec5b01c8e661af804e22d6)

Think it would be good if this description described the full effects of this option. Would maybe say "`NO_IPC`: don't build Cap’n Proto and libmultiprocess packages, and disable building IPC binaries (`bitcoin-node`, `bitcoin-gui`)"

Also, since this is the most significant commit in the PR, which controls whether IPC binaries are included in releases, it would be good to mention the effect this has on releases in the commit description.

Commit description could also mention effect this has on CI builds: it cause more depends builds to build IPC binaries, but still the only build running functional tests with them is the `i686_multiprocess` one.

Reply:
Expanded the comment and commit description.